### DISCUSSION TITLE
Trim Reblogs Scissors Not Appearing?

### DISCUSSION THREAD

**Discussion by klasssy** [AprilSylph/XKit-Rewritten#1009] [Help Desk]
I've been using XKit Rewritten for a few weeks and was so happy to use the trim reblogs function, but it now doesn't appear. I've tried using both regular posts and drafts and neither of them gives me the option to trim my reblogs. I've uninstalled and reinstalled the extension multiple times but nothing seems to work. Any advice is appreciated.

**Comment by marcustyphoon** [comment]
If you are on Chrome and you temporarily turn off every other script, then open a new Tumblr tab, does the button appear?

**Reply by klasssy** [reply]
Unfortunately not. I'm thinking it may be an issue on Tumblr's side as I've heard of heard others' having this problem right now.

**Reply by marcustyphoon** [reply]
If there's an experimental modification to the post footer on Tumblr's side, maaaybe? Does your quick tags button not appear either? If quick tags appears and trim reblogs does not, it's almost certainly our bug, but I'm not seeing an obvious reason why it would happen.

**Reply by marcustyphoon** [reply]
can I see a screenshot of one of the posts you are expecting to see the button on? there is one possibility here involving an unusual way to trim reblogs that some people do.

**Reply by klasssy** [reply]
> If there's an experimental modification to the post footer on Tumblr's side, maaaybe? Does your quick tags button not appear either? If quick tags appears and trim reblogs does not, it's almost certainly our bug, but I'm not seeing an obvious reason why it would happen.

Quick tags is appearing, so that's a bit odd. I ended up turning off the hide avatars function and it's now working!

**Reply by marcustyphoon** [reply]
Well those definitely have nothing to do with each other :D I am going to assume this is another occurrence of what we are affectionally calling "the haunted chromium bug" (well, I am) and hope that it's fixed by #945.

**Reply by rebelliouslegacy** [reply]
I'm having the same issue, and I have not found a fix for it; I hope this issue is fixed soon because it only shows up in my drafts folder but doesn't appear on my dash. I'm a role-playing blog, and doing replies have become tedious because I don't have the option to trim reblogs on the dash.
